src/components/Notifications.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, useEffect } from 'react';
+
+interface NotificationsProps {
+  onClose: () => void,
+  errorMessage: string,
+}
+
+export const Notifications: FC<NotificationsProps> = (props) => {
+  const { onClose, errorMessage } = props;
+
+  useEffect(() => {
+    setTimeout(() => {
+      onClose();
+    }, 3000);
```

Comment:
There is no `clearTimeout` in the return function of the `useEffect` hook. This can potentially lead to memory leaks or unexpected behavior if the component unmounts before the timeout has finished.